HGAlibs/HGAlibs2.py

```diff
@@ -0,0 +1,33 @@
+""" The Healing Gem of Arcadia Mad Libs Generator """
```

Comment:
When using comments, use a `#`, if you are not using the string.

Reply:
Aha! Got it.